js/locales/bootstrap-datepicker.my.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Malay translation for bootstrap-datepicker
+ * Nasrul Hazim <http://github.com/nasrulhazim>
+ */
+$.fn.datepicker.dates['my'] = {
```

Comment:
Shouldn't "ms" be the language code? See: https://en.wikipedia.org/?title=Malay_language right column at ISO 639-1